rules/aip0131/request_name_reference_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0131
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestRequestNameReference(t *testing.T) {
+	t.Run("Present", func(t *testing.T) {
+		f := testutils.ParseProto3String(t, `
```

Comment:
Can you rewrite these two into table tests, like what you did in the other PR? And also can you add a test for a negative case with field name not being "name"? Thank you!

Reply:
I added the not-name check (as a table test with absent). I am declining to write the "present" test as a table test because it gets sufficiently awkward to deal with the template with a multi-line annotation, and I find the existing test more readable.

Reply:
Fair enough! Thanks!